packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/edit/edit.html

```diff
@@ -752,6 +750,7 @@
                                     <span class="oui-radio__icon"></span>
                                     <span
                                         class="oui-radio__text"
+                                        style="max-width: 20rem"
```

Comment:
No other options ?

Reply:
adding a class ?
for "dirty" css like that i prefer inline style as it's straight forward when you read the code (and also it's difficult to find a relevent class name) but i'm not sure it's good practice, just personal preference. I can change it if necessary what do you think ?